src/ErrorHandler.php

```diff
@@ -265,7 +265,13 @@ public function addExceptionHandlerListener(callable $listener): void
      */
     private function handleError(int $level, string $message, string $file, int $line, ?array $errcontext = []): bool
     {
-        if (0 === (error_reporting() & $level)) {
+        // The error reporting `error_reporting()` will have when an error is captured that was silenced with the `@` operator
+        // is different on PHP 8 versus earlier versions where it's 0. On PHP 8 it defaults to errors that are not silencable.
+        $expectedSilencedErrorReportingLevel = \PHP_MAJOR_VERSION >= 8
```

Comment:
This value is not fixed, according to what I'm observing [here](https://3v4l.org/QGU0X), `error_reporting()` is returning the bitmask of all fatal error types  minus the ones that are excluded (`E_USER_ERROR` in that case)

Reply:
Jep I am seeing that too now, I'm trying to figure out why that didn't come up in testing.

But to drive the point home, this is still very much incorrect way to check if an error is silenced: https://3v4l.org/eQj7W.

I'm seeing if I can add a test for this and if I can resolve this point.